### PR TITLE
BB -12 merge into main developement

### PR DESCRIPTION
**Pull request by ZsomborHajzer**
Please review the code and point out any mistakes if seen. @2560p , @timothyadewale

### CODE REVIEW

**Comment by 2560p** [comment]
The code will have to be reviewed manually as due to the merge conflict. Please, await the upcoming comments.

File: CAR B-ZS/Distance_sensor/Distance_sensor.ino
Comment (2560p) [inline CAR B-ZS/Distance_sensor/Distance_sensor.ino:1]: Should the final version have or not have the additional files like this?

@ZsomborHajzer

File: CAR B-ZS/Gripper/Gripper.ino
Comment (2560p) [inline CAR B-ZS/Gripper/Gripper.ino:1]: Example code, might be removed before merging to `main`.

**Review by 2560p — APPROVED** [review]
@ZsomborHajzer, should we keep the sketch code in the main?

The pull request looks good, no major mistakes found.
Also need to fix the merge conflicts on the `B-Zs` branch, please, await the upcoming comments

File: CAR B-ZS/Distance_sensor/Distance_sensor.ino
Comment (ZsomborHajzer) [reply CAR B-ZS/Distance_sensor/Distance_sensor.ino:1]: No, I am OK with them being removed.

**Comment by 2560p** [comment]
The merge conflict resolving commit was issued. @timothyadewale, awaiting for the review!

File: CAR B-ZS/Distance_sensor/Distance_sensor.ino
Comment (timothyadewale) [reply CAR B-ZS/Distance_sensor/Distance_sensor.ino:1]: I reviewed the code, everything looks good to me.

**Review by timothyadewale — APPROVED** [review]

**Pull request merged**